src/components/AddColumnForm.vue

```diff
@@ -3,8 +3,8 @@
     class="cursor-pointer w-80 rounded-lg bg-white flex justify-center items-center gap-2 flex-col border-8 border-slate-300 hover:bg-slate-100"
     @click="showModal = true"
   >
-    <div class="text-3xl font-bold text-center">
-      <p class="text-6xl">+</p>
+    <div class="text-3xl font-bold text-center text-gray-500">
+      <p class="text-6xl text-gray-500">+</p>
```

Comment:
We don't need this if it is already defined in the parent, do we?
```suggestion
      <p class="text-6xl">+</p>
```

Reply:
true! thanks for finding it :)